src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+    private static final String NEW_LINE = System.lineSeparator();
+    private static final String VARIABLE1 = "supply";
+    private static final String VARIABLE2 = "buy";
```

Comment:
Краще не мати констант, чим мати константи з іменами VARIABLE1 та VARIABLE2 
```suggestion
    private static final String SUPPLY = "supply";
    private static final String BUY = "buy";
```